### PR TITLE
Added proper MRI entries in raisin bread for 21.0

### PR DESCRIPTION
**Pull request by cmadjar**
### Brief summary of changes

In this PR, the raisin bread dataset has been updated with proper imaging entries that actually make sense :). 

### This resolves issue...

- [x] Github https://github.com/aces/Loris/issues/4381

### To test this change...

- [ ] on a clean database, install raisin bread from scratch

### CODE REVIEW

File: raisinbread/RB_files/RB_MRICandidateErrors.sql [hunk 1]
```diff
@@ -1,29 +1,5 @@
 SET FOREIGN_KEY_CHECKS=0;
 TRUNCATE TABLE `MRICandidateErrors`;
 LOCK TABLES `MRICandidateErrors` WRITE;
-INSERT INTO `MRICandidateErrors` (`ID`, `TimeRun`, `SeriesUID`, `TarchiveID`, `MincFile`, `PatientName`, `Reason`) VALUES (1,'2016-07-27 18:33:24','1.3.12.2.1107.5.2.32.35177.30000011091713000714000003150',19,'/tmp/TarLoad-10-32-GaHdaJ/rom162_400162_v6_20110917_011745_3_mri.mnc','ROM162_400162_V6','Visit label does not exist');
```
Comment (cmadjar) [inline raisinbread/RB_files/RB_MRICandidateErrors.sql]: Note: the imaging pipeline does not insert anything anymore in this table, which is why it is empty.

Comment (driusan) [reply raisinbread/RB_files/RB_MRICandidateErrors.sql]: where does this class of errors get logged now?

Comment (cmadjar) [reply raisinbread/RB_files/RB_MRICandidateErrors.sql]: When uploading manually, it does not let you upload scans with different PSCID-DCCID.

When uploading using batch_upload_imagingupload.pl, you get an error in the log of the upload saying the PSCID & DCCID does not match (saved in the notification spool table I think).

**Comment by kongtiaowang** [comment]
Initialization of the database has an error.
insert this " docker-compose up --build " into .travis.yml to debug.

**Comment by cmadjar** [comment]
@kongtiaowang I added the line to .travis.yml as you suggested.

**Comment by ridz1208** [comment]
all thats missing for me is some documentation corrections and passing travis

**Comment by ridz1208** [comment]
potential changes:
```

ALTER TABLE mri_parameter_form 
ADD COLUMN  `fieldmapBOLD_Scan_Done` enum('yes','partial','no') DEFAULT NULL,
ADD COLUMN  `fieldmapBOLD_number_attempts` varchar(255) DEFAULT NULL,
ADD COLUMN  `fieldmapBOLD_comments` text,
ADD COLUMN  `t1_Scan_Done` enum('yes','partial','no') DEFAULT NULL,
ADD COLUMN  `t1_number_attempts` varchar(255) DEFAULT NULL,
ADD COLUMN  `t1_comments` text,
ADD COLUMN  `t2_Scan_Done` enum('yes','partial','no') DEFAULT NULL,
ADD COLUMN  `t2_number_attempts` varchar(255) DEFAULT NULL,
ADD COLUMN  `t2_comments` text,
ADD COLUMN  `fMRI_Scan_Done` enum('yes','partial','no') DEFAULT NULL,
ADD COLUMN  `fMRI_number_attempts` varchar(255) DEFAULT NULL,
ADD COLUMN  `fMRI_comments` text,
ADD COLUMN  `dwi25_Scan_Done` enum('yes','partial','no') DEFAULT NULL,
ADD COLUMN  `dwi25_number_attempts` varchar(255) DEFAULT NULL,
ADD COLUMN  `dwi25_comments` text,
ADD COLUMN  `dwi65_Scan_Done` enum('yes','partial','no') DEFAULT NULL,
ADD COLUMN  `dwi65_number_attempts` varchar(255) DEFAULT NULL,
ADD COLUMN  `dwi65_comments` text,
ADD COLUMN  `fieldmapDWI_Scan_Done` enum('yes','partial','no') DEFAULT NULL,
ADD COLUMN  `fieldmapDWI_number_attempts` varchar(255) DEFAULT NULL,
ADD COLUMN  `fieldmapDWI_comments` text;

UPDATE mri_parameter_form SET fieldmapBOLD_Scan_Done=(case FLOOR((RAND() * 3)) when 0 then 'yes' when 1 then 'partial' when 2 then 'no' end) WHERE Date_taken IS NOT NULL;
UPDATE mri_parameter_form SET t1_Scan_Done=(case FLOOR((RAND() * 3)) when 0 then 'yes' when 1 then 'partial' when 2 then 'no' end) WHERE Date_taken IS NOT NULL;
UPDATE mri_parameter_form SET t2_Scan_Done=(case FLOOR((RAND() * 3)) when 0 then 'yes' when 1 then 'partial' when 2 then 'no' end) WHERE Date_taken IS NOT NULL;
UPDATE mri_parameter_form SET fMRI_Scan_Done=(case FLOOR((RAND() * 3)) when 0 then 'yes' when 1 then 'partial' when 2 then 'no' end) WHERE Date_taken IS NOT NULL;
UPDATE mri_parameter_form SET dwi25_Scan_Done=(case FLOOR((RAND() * 3)) when 0 then 'yes' when 1 then 'partial' when 2 then 'no' end) WHERE Date_taken IS NOT NULL;
UPDATE mri_parameter_form SET dwi65_Scan_Done=(case FLOOR((RAND() * 3)) when 0 then 'yes' when 1 then 'partial' when 2 then 'no' end) WHERE Date_taken IS NOT NULL;
UPDATE mri_parameter_form SET fieldmapDWI_Scan_Done=(case FLOOR((RAND() * 3)) when 0 then 'yes' when 1 then 'partial' when 2 then 'no' end) WHERE Date_taken IS NOT NULL;


UPDATE mri_parameter_form SET fieldmapBOLD_number_attempts=(FLOOR((RAND() * 10))) WHERE Date_taken IS NOT NULL;
UPDATE mri_parameter_form SET t1_number_attempts=(FLOOR((RAND() * 10))) WHERE Date_taken IS NOT NULL;
UPDATE mri_parameter_form SET t2_number_attempts=(FLOOR((RAND() * 10))) WHERE Date_taken IS NOT NULL;
UPDATE mri_parameter_form SET fMRI_number_attempts=(FLOOR((RAND() * 10))) WHERE Date_taken IS NOT NULL;
UPDATE mri_parameter_form SET dwi25_number_attempts=(FLOOR((RAND() * 10))) WHERE Date_taken IS NOT NULL;
UPDATE mri_parameter_form SET dwi65_number_attempts=(FLOOR((RAND() * 10))) WHERE Date_taken IS NOT NULL;
UPDATE mri_parameter_form SET fieldmapDWI_number_attempts=(FLOOR((RAND() * 10))) WHERE Date_taken IS NOT NULL;



UPDATE candidate SET date_registered =(FROM_UNIXTIME(UNIX_TIMESTAMP('2016-07-15') - FLOOR(0 + (RAND() * 15768000))));
UPDATE candidate SET Date_active=date_registered;
```

File: raisinbread/README.md [hunk 2]
```diff
@@ -64,6 +64,21 @@ correct `host`, `url` and `base` values
...
+
+Note that for now, the files to download are only accessible to active members of the 
+MCIN lab using their LDAP credentials.
+
+1. Create a `/data/demo/data` directory where the files will go (make sure to have 
+21GB available on your disk)
+2. Run the following RSYNC command using your LDAP credentials:
+```
+rsync -av <myldapname>@ace-lab-1.acelab.ca:/home/users/shared/raisinbread/new_raisinbread_2019-05-30/ /data/demo/data/
+```
```
Comment (cmadjar) [inline raisinbread/README.md]: update documentation

**Comment by johnsaigle** [comment]
Needs PHPCS

>FILE: ...ld/aces/Loris/modules/mri_violations/test/mri_violationsTest.php
 508 | ERROR | [x] Line indented incorrectly; expected at least 8
     |       |     spaces, found 7
 509 | ERROR | [x] Line indented incorrectly; expected at least 8
     |       |     spaces, found 7
 510 | ERROR | [x] Line indented incorrectly; expected at least 8
     |       |     spaces, found 7
 511 | ERROR | [x] Line indented incorrectly; expected at least 8
     |       |     spaces, found 7
PHPCBF CAN FIX THE 4 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

File: raisinbread/README.md [hunk 3]
```diff
@@ -64,6 +64,12 @@ correct `host`, `url` and `base` values
 > The password of the `admin` user on the RB database is `demo20!7`
 
 
+#### Getting the imaging files
+MCIN members can get access to the imaging files by putting in a request on redmine
+to mount the /data-raisin bread folder with the imaging files on their VM.
+
+For external user, please email the loris-dev mailing list.
```
Comment (johnsaigle) [inline raisinbread/README.md]:
```suggestion
External users should email the loris-dev mailing list.
```

Comment (cmadjar) [reply raisinbread/README.md]: Done. Thanks for the suggestion!

**Comment by ridz1208** [comment]
@driusan if we can merge this on the sooner side, it would help us test imaging on test.loris.ca much better

File: raisinbread/README.md [hunk 4]
```diff
@@ -64,6 +64,12 @@ correct `host`, `url` and `base` values
 > The password of the `admin` user on the RB database is `demo20!7`
 
 
+#### Getting the imaging files
+MCIN members can get access to the imaging files by putting in a request on redmine
```
Comment (ridz1208) [inline raisinbread/README.md]: given what I read on redmine, people should automatically get access without putting in a request no ?

https://redmine.cbrain.mcgill.ca/issues/14481

File: raisinbread/README.md [hunk 5]
```diff
@@ -64,6 +64,12 @@ correct `host`, `url` and `base` values
...
+to mount the /data-raisin bread folder with the imaging files on their VM.
```
[the remark this answers is not in the document]
Comment (cmadjar) [reply raisinbread/README.md]: ooooops

File: raisinbread/README.md (hunk 4 above)
Comment (cmadjar) [reply raisinbread/README.md]: ooops. I updated the doc before Pam updated all the VM on Friday and forgot to update the doc accordingly. 

The README.md is updated in the next commit. Thanks!

File: raisinbread/README.md [hunk 6]
```diff
@@ -64,6 +64,12 @@ correct `host`, `url` and `base` values
 > The password of the `admin` user on the RB database is `demo20!7`
 
 
+#### Getting the imaging files
+MCIN members have automatically accessed to the imaging files on their dev VM
```
Comment (ridz1208) [inline raisinbread/README.md]:
```suggestion
MCIN members have automatic access to the imaging files on their dev VM
```

File: raisinbread/README.md [hunk 7]
```diff
@@ -64,6 +64,12 @@ correct `host`, `url` and `base` values
...
+where the raisinbread dataset is automatically mount in `/data-raisinbread`.
```
Comment (ridz1208) [inline raisinbread/README.md]:
```suggestion
where the raisinbread dataset is automatically mounted in the `/data-raisinbread` directory.
```

File: raisinbread/README.md [hunk 8]
```diff
@@ -64,6 +64,12 @@ correct `host`, `url` and `base` values
...
+
+External users should email the loris-dev mailing list.
```
Comment (ridz1208) [inline raisinbread/README.md]:
```suggestion
External users should email the loris-dev mailing list to request a copy of the data.
```

**Comment by cmadjar** [comment]
@ridz1208 Thank you for the great suggestions! They all have been integrated.

**Pull request merged**